sources/Inputs/include/Node.h

```diff
@@ -65,14 +166,18 @@ class Node {
    * @param id the node id
    * @param vl the voltage level element containing the node
    * @param nominalVoltage the nominal voltage of the node
+   * @param shunts the list of the shunts connectables to this node
    *
    * @returns the built node
    */
-  static std::shared_ptr<Node> build(const NodeId& id, const std::shared_ptr<VoltageLevel>& vl, double nominalVoltage);
+  static std::shared_ptr<Node> build(const NodeId& id, const std::shared_ptr<VoltageLevel>& vl, double nominalVoltage, const std::vector<Shunt>& shunts);
 
   const NodeId id;                                      ///< node id
   const std::weak_ptr<VoltageLevel> voltageLevel;       ///< voltage level containing the node
   const double nominalVoltage;                          ///< Nominal voltage of the node
+  const std::vector<Shunt> shunts;                      ///< Shunts connectables to the node
+  std::vector<std::weak_ptr<Line>> lines;               ///< Lines to which the node belong
```

Comment:
"Lines connected to this node" and similarly for transfos

Reply:
OK